release.nix

```diff
@@ -1,9 +1,23 @@
-let self = import ./. { localSystem.system = "x86_64-linux"; };
-    lib = self.pkgs.lib;
-in
-  {
-    generic-cli = self.alamgu.generic-cli;
-  }
-  // lib.mapAttrs' (n: lib.nameValuePair ("nanos--" + n)) self.nanos
-  // lib.mapAttrs' (n: lib.nameValuePair ("nanox--" + n)) self.nanox
-  // lib.mapAttrs' (n: lib.nameValuePair ("nanosplus--" + n)) self.nanosplus
+let
+  alamgu-path = import ./dep/alamgu/thunk.nix;
+  pkgsSrc = import (alamgu-path + "/dep/nixpkgs/thunk.nix");
+  lib = import (pkgsSrc + "/lib");
+
+  x86_64-linux = import ./. rec {
```

Comment:
```nix
systems = lib.genAttrs [ "x86_64-linux" "x86_64-darwin" ] (system: import ./. {
   localSystem = { inherit system; };
};
```
Preferred in case we update this to support more than x86_64-* host archs, makes it a lot easier to add them!

Reply:
Done!